### PR TITLE
Added Preview 3 changelog

### PR DESCRIPTION
**Pull request by Sergio0694**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# API changes in Preview 3:
+
+🆕 New `ObservableValidator` class, which supports the [`INotifyDataErrorInfo`](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifydataerrorinfo) interface.
+
+🆕 Added new constructors with cancellation support to the async commands, and added new cancellation-related properties to the async command interfaces.
+
+🆕 Added a new `WeakReferenceMessenger` type. This type is less performant than the other messenger, and uses more memory, and in return only uses weak references to track recipients. This type essentially mirrors the behavior of the `Messenger` type from `MvvmLight`, making the transition easier for developers migrating from that library.
+
+🆕 Introduced a new custom delegate to represent message handlers, which also receives the current recipient as additional input parameter. No API changes are required for users registering messages through the `IRecipient<TMessage>` interface, whereas users manually registering will need to modify their code as follows:
```
Comment (michael-hawker) [inline CHANGELOG.md:9]: Do we want to mark where there are required code changes, like here and the SetPropertyAndNotifyOnCompletion below?

Comment (Sergio0694) [reply CHANGELOG.md:9]: I do indicate the changes in the code snippet right after this point, do you mean something else? 🤔

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    set => SetProperty(() => user.Name, value);
+
+    // Preview 3
+    set => SetProperty(user.Name, value, user, (u, n) => u.Name = n);
+}
+```
+
+The syntax is slightly more complex, but results in a 150x speed improvement (that's not a typo), requires no memory allocations at all and no reflection, and ensures that all necessary validation of the arguments can be done at compile time too.
+
+✅ API changes to the `SetPropertyAndNotifyOnCompletion` (as detailed in the blog post).
```
Comment (michael-hawker) [inline CHANGELOG.md:56]:
```suggestion
✅ API changes to the `SetPropertyAndNotifyOnCompletion` (as detailed in [this blog post]( https://devblogs.microsoft.com/pax-windows/mvvm-toolkit-preview-3-the-journey-of-an-api/)).
```

File: CHANGELOG.md (hunk 1 above)
Comment (Sergio0694) [reply CHANGELOG.md]: Reorganized that section in e954634ed26aef9f4cd1e8815812325f7da7af4f, let me know what you think! 😊

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+🆕 Introduced a new custom delegate to represent message handlers, which also receives the current recipient as additional input parameter (see code changes below).
+
+✅ Renamed `Messenger` to `StrongReferenceMessenger`.
+
+✅ The `WeakReferenceMessenger` is now the default messenger used by the `ObservableRecipient` class.
+
+✅ Changed `ObservableObject` overloads using `Expression<Func<T>>` to be more efficient (see code changes below).
+
+✅ API changes to the `SetPropertyAndNotifyOnCompletion` (as detailed in [this blog post]( https://devblogs.microsoft.com/pax-windows/mvvm-toolkit-preview-3-the-journey-of-an-api/)).
```
Comment (michael-hawker) [inline CHANGELOG.md:17]: This and the IoC one need to be in the section below as well. 🙂

Comment (Sergio0694) [reply CHANGELOG.md]: Added more info in https://github.com/windows-toolkit/MVVM-Samples/pull/23/commits/5244c220822add0d95d35d78149aaeff30833765 and also linked to the preview docs in my other branch about the migration for the `Ioc` class 😄

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**